phoenix-core/src/main/java/org/apache/phoenix/index/IndexMaintainer.java

```diff
@@ -964,8 +964,7 @@ public Put buildUpdateMutation(KeyValueBuilder kvBuilder, ValueGetter valueGette
             // add the keyvalue for the empty row
             put.add(kvBuilder.buildPut(new ImmutableBytesPtr(indexRowKey),
                 this.getEmptyKeyValueFamily(), dataEmptyKeyValueRef.getQualifierWritable(), ts,
-                // set the value to the empty column name
```

Comment:
Are there any tests to validate this empty column value? If not, perhaps we should add one.